app/src/state/actions/activity/Activity.ts

```diff
@@ -67,6 +67,12 @@ interface IGetSuccess {
 interface IActivityFormChangeRequest {
   eventFormData: UserRecord;
 }
+
+interface SwitchRecordSetPayload {
+  setId: number;
```

Comment:
```suggestion
  setId: string;
```
records 1-4 are '1', '2', '3', '4', custom recordsets are nanoids (once upon a time they were date objects